about/policy/sgid/index.md

```diff
@@ -10,16 +10,33 @@ categories: []
 date: 2019-12-04 10:01:27 -0500
 ---
 
-The following applies to data in the internal SGID database.
+The following policies apply to data in the internal SGID geodatabase, which is a curated database of state-wide datasets. It is maintained as an internal, AGRC-only Esri Enterprise Geodatabase and is available to everyone as the cloud-hosted, PostGIS-backed [OpenSGID]({% link sgid/open-sgid/index.md %}). The contents of the SGID are also uploaded to the Utah SGID groups of our [ArcGIS Online Organization](https://utah.maps.arcgis.com).
 
-## Adding New Tables
+## Adding New Datasets to the SGID
```

Comment:
**thought**: what about generalizing this to talk about the SGID and not just adding data to it?
```suggestion
## SGID Data
```